Dynatrace/dynatrace-operator
# Use -s param instead of "partial: ..." to generate partial manifests using Helm

## Description

[K8S-9602](https://dt-rnd.atlassian.net/browse/K8S-9602)

We want to generate only given parts of our Helm-chart for the release, and we used to do it by setting a "secret" param called `partial`, which led to a very overcomplicated and confusing logic.

I replaced it with using the `-s` (`--show-only`) in the makefiles instead.
[official docu:](https://helm.sh/docs/helm/helm_template/)
```
 -s, --show-only stringArray                      only show manifests rendered from the given templates
```

Sidenote: I tried to make it "prettier" by trying not to list each CSI-related file 1-by-1, but didn't find a way

## How can this be tested?
Try out all helm/manifests related commands 😅 (no change in behaviour should be expected)
(Important to check not only the directly touched make commands, because all yaml files were touched do to the removal of the `partial` check)
